api/models/document.js

```diff
@@ -2,40 +2,79 @@
 
 require('./file');
 
+const helpers = require('../helpers');
 const bookshelf = require('../../config').bookshelf;
 const BaseModel = require('./base');
+const relatedModels = [
+  'file',
+  'trials',
+]
 
 const Document = BaseModel.extend({
   tableName: 'documents',
+  outputVirtuals: false,
   visible: [
-    'type',
+    'id',
+    'source_id',
     'name',
-    'source_url',
+    'type',
+    'trials',
+    'file',
+    'url',
   ],
-  file: function () {
-    return this.belongsTo('File');
-  },
   serialize: function (options) {
     const attributes = Object.assign(
       {},
       Object.getPrototypeOf(Document.prototype).serialize.call(this, arguments)
     );
 
+    attributes.url = this.url;
+    attributes.trials = (this.relations.trials || []).map((trial) => trial.toJSONSummary());
+
+    if (this.relations.file) { 
+      attributes.file = this.relations.file.toJSONSummary(); }
+    else {
+      attributes.file = {};
+    }
+
+    return attributes
+  },
+  file: function () {
+    return this.belongsTo('File');
+  },
+  trials: function () {
+    return this.belongsToMany('Trial', 'trials_documents');
+  },
+  toJSONSummary: function () {
+    const attributes = this.attributes;
```

Comment:
I used `this.attributes` here because I needed access to `source_url` which is not visible and thus not available through `toJSON()`.